vsintegration/src/FSharp.Editor/Hints/Hints.fs

```diff
@@ -17,6 +19,7 @@ module Hints =
             Kind: HintKind
             Range: range
             Parts: TaggedText list
+            GetTooltip: Document -> CancellationToken -> Async<TaggedText list>
```

Comment:
If we're using `Async` then passing cancellation token explicitly shouldn't be needed.